src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -65,15 +65,13 @@ public void parse_configFileOnly_success() throws ParseException, IOException {
         //Optional arguments have default values
         Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
         Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertTrue(Files.isSameFile(Paths.get("."), cliArguments.getOutputFilePath()));
```

Comment:
Can we change the default output file path test as checking if the file name format is the same as `REPORT_NAME_FORMAT`?

Reply:
I am using the following  to check if the output folder is in yyyy-MM-dd-HH-mm-ss format
        
```Constants.DEFAULT_REPORT_NAME_FORMAT.parse(cliArguments.getOutputFilePath().getFileName().toString());```